sources/corepackUtils.ts

```diff
@@ -137,16 +148,26 @@ export async function installVersion(installTarget: string, locator: Locator, {s
 
   stream.pipe(sendTo);
 
-  const hash = build[0]
-    ? stream.pipe(createHash(build[0]))
-    : null;
-
+  const hash = stream.pipe(createHash(build[0] ?? `sha256`));
```

Comment:
I suggest we use `sha224` because it's shorter
```suggestion
  const hash = stream.pipe(createHash(build[0] ?? `sha224`));
```

Reply:
I'd have a small preference for sha256 because it's more well-known. I suspect most people seeing sha224 will wonder whether it's less secure and perhaps document themselves about it. sha256 seems less friction for users.

Reply:
It's easy to change if folks complain about it, but I suspect most folks won't pay attention to it – unless they're into software security, in which case I have the feeling they'd know what sha224 is. Until anyone complains `sha224` seems a better choice for me, no strong feelings though

Reply:
> sha224 seems a better choice for me

Can you explain why?

Reply:
Because it's shorter, and more than strong enough for what that hash is used for.

Reply:
My intuition is that the extra characters won't change a lot the experience, but sha256 is more well-known. I kept it as-is for this iteration.